Developer_Manual.rst

```diff
@@ -4590,7 +4590,7 @@ dropping non interesting, or rewriting exceptions.
 Caching of Python level compilation
 ===================================
 
-While the C compilation result is already cached with ccache and friends
+While the C compilation result is already cached with `ccache` and friends
```

Comment:
The quoting in rest should be double back-ticks, it's easy to make that mistake as it's different from markdown, otherwise it's good.